src/OrbitQt/orbitglwidget.cpp

```diff
@@ -255,31 +155,33 @@ void OrbitGLWidget::mouseReleaseEvent(QMouseEvent* event) {
   update();
 }
 
-void OrbitGLWidget::showContextMenu() {
+void OrbitGLWidget::ShowContextMenu() {
```

Comment:
Context menu support here has only been needed for the ImGui debug widget and will be removed in a subsequent PR.